schema.sql

```diff
@@ -8,4 +8,26 @@ CREATE TABLE animals (
  escape_attempts INT NOT NULL,
  weight_kg DECIMAL NOT NULL);
 
-ALTER TABLE animals ADD column species VARCHAR(100); 
+/*species table*/
+CREATE TABLE species (
+species_id INT GENERATED ALWAYS AS IDENTITY,
+name VARCHAR(100) NOT NULL,
+PRIMARY KEY(species_id));
+
+ALTER TABLE owners ADD age INT NOT NULL;
+ALTER TABLE animals RENAME species_id TO species;
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id;
+ALTER TABLE animals ADD COLUMN INT;
+
+
+/*owners table*/
+CREATE TABLE owners(
+full_name VARCHAR(100) NOT NULL,
+owners_id INT GENERATED ALWAYS AS IDENTITY,
+PRIMARY KEY(owners_id));
```

Comment:
- [ ] Please add `age` column in this table

Reply:
done